source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,37 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-06-30
+
+### Highlights
+
+#### Emoji Enhancements with Skin Tone Selection
+ - Added support for emoji standard v13.0. Users will see Reactions and Emoji rendered based on their operating system or device, and have the ability to choose various skin tones using the Mattermost emoji picker.
```

Comment:
> Emoji rendered based on their operating system or device

I don't know if this is true? Asked here: https://community-daily.mattermost.com/private-core/pl/gk4b7i5esfn3bxgzept7xe79ye

Reply:
Removed that part.